iontestdata/good/equivs/localSymbolTableAppend.ion

```diff
@@ -0,0 +1,80 @@
+// Tests several uses of appending symbols to the current local symbol table context.
+
+embedded_documents::[
+    "s1 s2 s1 s2 s3 s4 s5",
+
+    // The basic case.
+
+    '''$ion_symbol_table::{'''
```

Comment:
The other files in this PR have a single set of ''' per embedded_document. Having the long string triple-quotes on each line introduces a lot of visual noise -- is there a reason to prefer concatenation here?

Reply:
Nope.